src/components/Compare/CompareTableRow.tsx

```diff
@@ -59,17 +71,30 @@ const CompareTableRow = (props: {
     ? location.locationInfo.county.replace('Parish', 'Par.')
     : location.locationInfo.county.replace('County', 'Co.');
 
+  const roundedPopulation = (
+    Math.round(location.locationInfo.population / 100) * 100
+  ).toLocaleString();
+
   return (
     <Row
       index={location.rank}
       isCurrentCounty={isCurrentCounty}
       onClick={handleLocationClick}
       isModal={isModal}
     >
-      <MetricCell iconColor={location.metricsInfo.level}>
-        <span>{location.rank}</span>
-        <FiberManualRecordIcon />
-        {locationName}
+      <MetricCell
+        iconColor={location.metricsInfo.level}
+        sortByPopulation={sortByPopulation}
+      >
+        <div>
+          <span>{location.rank}</span>
+          <FiberManualRecordIcon />
+        </div>
+        <div>
+          {locationName}
+          <br />
+          <Population>{roundedPopulation}</Population>
```

Comment:
It would be nice to align the population to the right somehow, but I'm not sure if it would look better since the location names are aligned to the left.

Reply:
will bring up to ux